src/components/KeyValueEditor/KeyValueEditor.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+
+import { KeyValueEditor } from './KeyValueEditor';
+
+const dataArray = [
```

Comment:
Is declared in multiple files, could be exported in a single constants file or in one of the files. Maybe look at how it's currently done in FlowPlayer component so we can try and keep consistency.

Reply:
ik had dit ooit al een geprobeerd om mocks te importeren in de test vanuit de story file, maar dat werkt niet.
Importeren vanuit een 3de mock file lukt wel :D
Thx for the tip